<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomsList</class>
 <widget class="QMainWindow" name="RoomsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="newRoomLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Название комнаты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newRoomPushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить комнату</string>
    </property>
   </widget>
   <widget class="QListWidget" name="roomsListWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>630</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
